### ISSUE TITLE
`--open` flag not working in Parcel 2

### ISSUE THREAD

**Issue by pnfcre** [parcel-bundler/parcel#3803]
# 🐛 Bug Report

I tried passing `--open` to the `parcel src/index.html` command, but nothing happened. This feature works perfectly in the latest stable `parcel-bundler` package.

## 🎛 Configuration (package.json)

```json
{
  "scripts": {
    "start": "parcel src/index.html --open"
  }
}
```

## 🤔 Expected Behavior

Parcel should open `http://localhost:1234` in my default web browser.

## 😯 Current Behavior

Nothing happens.

## 🔦 Context

I tried to use the `--open` flag as I'm used to do in `parcel-bundler`.

## 💻 Code Sample

src/index.html:
```html
hello world
```

package.json:
```json
{
  "scripts": {
    "start": "parcel src/index.html --open"
  }
}
```

```sh
$ yarn start
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-alpha.2.1
| Node             | v13.0.1
| Yarn             | 1.19.1
| Operating System | Windows 10 v1809 (x64)

**Comment by Taym95** [comment]
I will take this issue!

**Comment by mackness** [comment]
Seem to work with parcel@1.12.4

**Comment by pnfcre** [comment]
Yes, this issue is specific to Parcel 2 @mackness.

**Comment by Taym95** [comment]
I think this one can be closed! #4596

**Comment by DeMoorJasper** [comment]
Ow yes it can, thanks for the heads up
